rfc/0054-sdk3-management-apis.md

```diff
@@ -342,15 +342,26 @@ Iterable<QueryIndex> GetAllIndexes(string bucketName, [options])
 * Optional:
 
   * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
+  * `CollectionName` - the name of the collection to restrict indexes to.
+  * `ScopeName` - the name of the scope to restrict indexes to.
+    * If either `CollectionName` or `ScopeName` are set then both *must* be set.
 
 ### N1QL
 
+No collections set:
 ```
 SELECT idx.* FROM system:indexes AS idx
 WHERE keyspace_id = "bucketName" AND `using`="gsi"
```

Comment:
It is going to pull collection aware indexes too, right? Because we are matching by bucket name only. Is this expected behavior? Should we document it somehow explicitly?

Reply:
I meant could it be recorded explicitly in the RFC? Right now it says

> Fetches all indexes from the server.